apps/src/templates/teacherDashboard/SectionRow.jsx

```diff
@@ -245,6 +245,15 @@ class SectionRow extends Component {
 
     const persistedSection = !!section.code;
 
+    let sectionCode = '';
+    if (!editing) {
+      if (section.providerManaged) {
+        sectionCode = '---';
+      } else {
+        sectionCode = section.code;
+      }
+    }
```

Comment:
I'd like to update the '---' with a little `fa-question-circle` icon and a tooltip explaining why the section doesn't have a code.  Maybe not needed...